guides/common/modules/proc_configuring-a-host-to-use-the-pull-client.adoc

```diff
@@ -4,11 +4,20 @@
 For {SmartProxies} configured to use `pull-mqtt` mode, hosts can subscribe to remote jobs using the remote execution pull client.
 Hosts do not require an SSH connection from their {SmartProxyServer}.
 
+The `katello-pull-transport-migrate` package installs `foreman_ygg_worker` and Yggdrasil packages as dependencies, configures the Yggdrasil client, and starts the pull client worker on the host.
+
 .Prerequisites
 * You have registered the host to {Project}.
 * The {SmartProxy} through which the host is registered is configured to use `pull-mqtt` mode.
 For more information, see {InstallingSmartProxyDocURL}configuring-remote-execution-for-pull-client_{smart-proxy-context}[Configuring pull-based transport for remote execution] in _{InstallingSmartProxyDocTitle}_.
 include::snip_prerequisite-project-client-repository-enabled.adoc[]
+ifdef::foreman-el[]
+* The {client-clevis-repository} repository is available for the host.
```

Comment:
Please introduce a different attribute for the "AppStream" repo.

Reply:
Actually, we seem to be fine with just "AppStream" elsewhere: https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/proc_installing-standalone-hammer.adoc?plain=1  I think it should work here too.